command/agent.go

```diff
@@ -536,6 +563,12 @@ func (c *AgentCommand) Run(args []string) int {
 		}
 	}()
 
+	// Wait for the template to render
+	select {
+	case <-ctx.Done():
+	case <-ts.UnblockCh:
```

Comment:
Would this block forever if `ts` is not initialized?

Reply:
It would. I suppose that would happen if no auth-auth / method was setup. Fixed in 419f21b95